lib/SearchAndSort/SearchAndSort.js

```diff
@@ -1141,12 +1141,17 @@ class SearchAndSort extends React.Component {
             onMarkPosition={resultsOnMarkPosition}
             onMarkReset={resultsOnResetMarkedPosition}
             itemToView={resultsCachedPosition}
-            isSelected={({ item, criteria }) => isMatch(item, criteria)}
+            isSelected={({ item, criteria }) => {
+              if (criteria) {
+                return isMatch(item, criteria);
+              }
+              return false;
+            }}
           />
         )}
       </FormattedMessage>
     );
-  }
+  }aaaaaa
```

Comment:
@JohnC-80 I like this :)

Reply:
Sometimes I just have to get it all out.